tests/conftest.py

```diff
@@ -129,9 +130,12 @@ def testbed_devices(ansible_adhoc, testbed):
     #       from common.devices import FanoutHost
     #       devices["fanout"] = FanoutHost(ansible_adhoc, testbed["dut"])
 
+    vm_base = int(testbed['vm_base'][2:])
+    devices['neighbor'] = {}
+    for k, v in testbed['topo']['properties']['topology']['VMs'].items():
+        devices['neighbor'][k] = EosHost(ansible_adhoc, "VM%04d" % (vm_base + v['vm_offset']), creds['eos_login'], creds['eos_password'])
```

Comment:
Could you please create a dedicated fixture for the eos host? My original idea of creating this `testbed_devices` was not good. It could get bigger and bigger like this. Some test scripts using this fixture may only need access to duthost. Then initializing other objects is a waste and causes overhead for running these tests. Probably I'll create a PR to deprecate the `testbed_devices` fixture in the future.

Reply:
I was mainly following comment from 127-131. is there a way for the test to indicate it will use eos host or not?

also, this is only to init the ansible host. it will not contact the eos host.

Reply:
moved to nbrhost fixture. i'll let you to deprecate testbed_devices.

Reply:
Thanks! It would be easier for test scripts to declare required fixtures on needed basis than specifying somewhere whether it will use eos host or not.